src/typing/typeloadModule.ml

```diff
@@ -744,12 +744,13 @@ let init_module_type ctx context_init do_init (decl,p) =
 		let ctx = { ctx with type_params = a.a_params } in
 		let is_type = ref false in
 		let load_type t from =
+			let _, pos = t in
```

Comment:
Would be a bit more idiomatic to have `let pos = snd t in`, but this is fine too.